content/chainguard/chainguard-images/working-with-images/scanners/grype-tutorial/index.md

```diff
@@ -0,0 +1,250 @@
+
+grype is appropriate for one-off detection for manual CVE mitigation and in automated use in CI pipelines. Chainguard maintains a [low-to-no CVE Chainguard Image for grype](https://edu.chainguard.dev/chainguard/chainguard-images/reference/grype/) based on our lightweight Wolfi distribution.
+
+## Installation
+
+### Container Images
+
+grype is readily available as a container image. To pull the low-to-no-CVE Chainguard Image for grype and perform a scan on the official Docker nginx image, run the following:
+
+```bash
+docker run -it cgr.dev/chainguard/grype nginx
+```
+
+Alternatively, you can scan using the official grype Docker image:
+
+```
+docker run -it anchore/grype:latest nginx
+```
+
+### Binary Installation
+
+grype provides an installation script. To use it, change the path following the `-b` flag to a preferred installation location on your system path, such as `/usr/bin`.
+
+```
+curl -sSfL https://raw.githubusercontent.com/anchore/grype/main/install.sh | sh -s -- -b /usr/local/bin
+```
+
+Alternately, find and download the appropriate package file or binary from the [official releases page](https://github.com/anchore/grype/releases). Place the binary on your system path or install the package using the conventions for your OS and distribution.
+
+grype can also be [built from source](https://github.com/anchore/grype/blob/main/DEVELOPING.md#native-development).
+
+### Install via Package Manager
+
+grype can be installed using the following commands specific to your OS and distribution: 
+
+#### Homebrew
+
+```
+brew tap anchore/grype
+brew install grype
+```
+
+#### Chocolatey
+
+```
+choco install grype -y
+```
+
+## Basic Usage
+
+Throughout this tutorial, we'll use the `grype` command to run grype. If you're running grype asacontainer image, replace this command with the appropriate `docker run` command, such as `docker run -it cgr.dev/chainguard/grype`.
+
+To run grype on an image in a Docker or OCI registry, pass the image name as an argument:
+
+```bash
+grype nginx
+```
+
+```bash
+grype cgr.dev/chainguard/nginx
+```
+
+To scan an image stored to `.tar`, pas the path to the archive file as an argument:
+
+```bash
+docker pull cgr.dev/chainguard/nginx
+docker save cgr.dev/chainguard/nginx > nginx_chainguard_image.tar.gz
+grype nginx_chainguard_image.tar.gz
+```
+
+grype can scan local directories, such as Python virtual environment (venv) or `node_modules` folders.
+
+
+```bash
+python -m venv venv
+./venv/bin/pip install WTForms==2.3.3 Werkzeug==2.0.1
+grype venv
+```
+
```

Comment:
Again, I'd break up the context and separate it out for each of the relevant code blocks

Reply:
OK, since this made that section longer, I added new headings for, i.e. image on registry, directory, tar, SBOM. Wrote explanatory lines for each operation (creating folders, etc).

Reply:
Ah, I misread your comment, but think the section is clearer now with more explanation and headings. Went in a direction of more subheadings and more explanation for the direcotry subheading. I think this is a page likely to be scanned and/or excerpted in results, so I like the heading approach for easy parsing.